### ISSUE TITLE
recommendations for additional restructuring

### ISSUE THREAD

**Issue by pssachdeva** [dlab-berkeley/R-Machine-Learning-Legacy#55]
I delivered the workshop with these current materials, and things went pretty smoothly, but it's too much material for two 3 hour workshops.

I'd recommend splitting this workshop into either 3 or 4 2-hour workshops. This could look something like:
- Part 1: introduction and regression
- Part 2: preprocessing
- Part 3: regularization and cross-validation
- Part 4: more models

Perhaps Parts 1/2 could be merged into a single workshop. 

I'd also recommend beefing up the "more models" section to more appropriately fill a 2 hour slot: more details on logistic regression, naive bayes, and random forest

**Comment by asteves** [comment]
All good suggestions. Happy to talk further on implementation once the labor action is complete.
